crates/task-impls/src/quorum_proposal/dependency_handle.rs

```diff
@@ -175,6 +177,30 @@ impl<TYPES: NodeType> HandleDepOutput for ProposalDependencyHandle<TYPES> {
 
     #[allow(clippy::no_effect_underscore_binding)]
     async fn handle_dep_result(self, res: Self::Output) {
+        let high_qc_view_number = self.consensus.read().await.high_qc().view_number;
```

Comment:
Followup on an earlier discussion, in all testing, this value, and the provided high qc value were never different, so it seemed like a red herring. I can add the change in though that effectively takes the high qc from the event if it's available, and falls back to the shared state otherwise.